expressions.go

```diff
@@ -319,6 +319,42 @@ var functions = map[string]struct {
 			return nil, fmt.Errorf("sum expects argument #1 to be slice of numbers")
 		}
 	}},
+	"sumiftrue": {2, func(args []Value) (Value, error) {
+		values, ok := args[0].(*Slice)
```

Comment:
args[0] and args[1] could be `undefined` themselves if they were never ever set

(One could question the value of this behavior, vs having an empty slice in this case!)